<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Notification_Menu</class>
    <widget class="QWidget" name="Notification_Menu">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>760</width>
                <height>680</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QPushButton" name="start_popups_button">
            <property name="geometry">
                <rect>
                    <x>80</x>
                    <y>150</y>
                    <width>250</width>
                    <height>47</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Start Pop-Ups</string>
            </property>
        </widget>
        <widget class="QPushButton" name="stop_popups_button">
            <property name="geometry">
                <rect>
                    <x>430</x>
                    <y>150</y>
                    <width>250</width>
                    <height>47</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Stop Pop-Ups</string>
            </property>
        </widget>
        <widget class="QPushButton" name="apps_button_menu">
            <property name="geometry">
                <rect>
                    <x>255</x>
                    <y>330</y>
                    <width>250</width>
                    <height>47</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Apps</string>
            </property>
        </widget>
        <widget class="QPushButton" name="exit_button">
            <property name="geometry">
                <rect>
                    <x>330</x>
                    <y>500</y>
                    <width>100</width>
                    <height>35</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Exit</string>
            </property>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>